src/mobilityPackagePriceEstimation/java/org/opentripplanner/estimator/utils/RandomLocationGenerator.java

```diff
@@ -0,0 +1,42 @@
+package org.opentripplanner.estimator.utils;
+
+import org.geotools.measure.Latitude;
+import org.geotools.measure.Longitude;
+import org.opentripplanner.common.model.GenericLocation;
+
+import java.util.Random;
+
+public class RandomLocationGenerator {
+
+    private Random random;
+
+    public RandomLocationGenerator(Random random) {
+        this.random = random;
+    }
+
+    public GenericLocation generateRandomLocation(GenericLocation center, double radius) {
```

Comment:
I am not 100% sure, but I think this code doesn't generate random location with even distribution

Reply:
I looked it up and it is actually a proper algorithm - nice!